src/database/manager.ts

```diff
@@ -0,0 +1,155 @@
+import { DB_CLUSTER_ENV, DB_NAME_ENV, DB_SECRET_ENV } from "../cdk/generator"
+import {
+  Connection,
+  ConnectionManager,
+  ConnectionOptions,
+  createConnection,
+  EntitySchema,
+  getConnectionManager,
+} from "typeorm"
+import { LoggerOptions } from "typeorm/logger/LoggerOptions"
+
+// todo: maybe make connection names match DB names
+// to support multiple connections to different databases
+const CONNECTION_NAME = "default"
+
+export interface ConnectionOptionsOverrides extends Omit<ConnectionOptions, "type"> {
+  database?: string
+}
+
+export interface DatabaseManagerProps extends ConnectionOptionsOverrides {
+  printQueries?: boolean | undefined
```

Comment:
you are doing the same here

Reply:
<img width="727" alt="Screen Shot 2021-06-25 at 8 17 32 PM" src="https://user-images.githubusercontent.com/245131/123461934-62200680-d5f2-11eb-83d5-93647970fe1d.png">

It doesn't make sense to me either

Reply:
I reinstalled vscode and the error went away. wacky 🤷🏻